modules/Microsoft.DBforPostgreSQL/flexibleServers/configurations/deploy.bicep

```diff
@@ -0,0 +1,51 @@
+@description('Required. The name of the database.')
```

Comment:
Rather the name of the configuration,  I guess

Reply:
Good pickup. Fixed